src/info/mod.rs

```diff
@@ -335,7 +335,7 @@ impl Info {
                 languages.push((
                     "Other".to_string(),
                     other_perc,
-                    DynColors::Ansi(AnsiColors::White),
+                    DynColors::Ansi(AnsiColors::Default),
```

Comment:
`on_color()` with `Default` uses the default Terminal Bg color which makes it invinsible.

Reply:
Oh, sorry about that. That should definitely be kept as `White` then.

Reply:
Alternatively block characters can be used to take advantage of the foreground color, but that might have less compatibility among terminals.

Reply:
Or perhaps it's OK if the block is invisible? It should always be a negligibly small percentage, after all?

Reply:
Not necessarily, let's say we have a repo of 10 languages with an equal distribution then `other` would account for ~50%. Besides, this is what Github does.

Reply:
Good point, I didn't consider that. Unless we're using block characters, I think we'll have to switch back to `::White`, then.

I think white BG would be visible enough even on a light terminal, since there's no "alpha channel" like there is with ASCII characters. So I think just switch back to `::White` for that, what do you think?